pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[tool.poetry]
+name = "magma-suite"
+version = "1.0.0b3"
+description = "Collection of tools to manage meta-workflows automation."
+authors = ["Michele Berselli <berselli.michele@gmail.com>", "Doug Rioux", "Soo Lee", "CGAP team"]
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/dbmi-bgm/magma"
+homepage = "https://github.com/dbmi-bgm/magma"
+classifiers = [
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: POSIX :: Linux',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3'
+]
+packages = [
+    { include="magma" },
+    { include="magma_ff" }
+]
+
+
+[tool.poetry.dependencies]
+python = ">=3.6.1,<3.10"
+dcicutils = ">=3.5.0"
```

Comment:
we need to double check these dependencies are all and up to date with the latest releases, I know Kent and Will have been updating those a lot